src/pages/Loans/LoansOverview/components/LoansMarkets/MarketTable.tsx

```diff
@@ -0,0 +1,29 @@
+import { useId } from '@react-aria/utils';
+
+import { Card, Stack, Table, TableProps } from '@/component-library';
+
+import { StyledMarketTitle } from './LoansMarkets.style';
+
+type Props = {
+  title: string;
+};
+
+type InheritAttrs = Omit<TableProps, keyof Props>;
+
+type MarketTableProps = Props & InheritAttrs;
+
+const MarketTable = ({ title, ...props }: MarketTableProps): JSX.Element => {
+  const titleId = useId();
```

Comment:
is there any way to use a meaningful id here, rather than a randomly generated one?

Reply:
Yes, you can pass a id as you want but it needs to be unique.

Reply:
yes - question is whether there's anything you're passing in to the component that you can use as an id, rather than generating one? Not a big deal but makes things a bit tidier.

Reply:
For now, I dont think so.